.travis.pylintrc

```diff
@@ -0,0 +1,378 @@
+[MASTER]
```

Comment:
Do we need this file at the top level?
Why is it prefixed with ".travis"?
Where are the settings from?

Reply:
```
pylint --generate-rcfile > .pylintrc
```

it's just been an example to test running pylint against with a given config file